fluent-syntax/src/util.js

```diff
@@ -0,0 +1,3 @@
+export function includes(arr, elem) {
+  return arr.indexOf(elem) > -1;
```

Comment:
We could change `String.prototype` and `Array.prototype` and somehow do this only for the compat build. I don't think it's worth it though. Almost all of our needs are covered by the current pipeline. `includes` is a rare exception.